src/pushsource/_impl/source.py

```diff
@@ -77,7 +77,11 @@ def __iter__(self):
 
         generator = self.__delegate.__iter__()
         for item in generator:
-            if not hasattr(item, "src") or not item.src:
+            if (
+                not hasattr(item, "src")
+                or not item.src
+                or not item.src.startswith("/mnt")
```

Comment:
it would be nice to have this configurable or maybe check just for  /

Reply:
Changed to /